src/main/java/net/minecraftforge/fml/hooks/BasicEventHooks.java

```diff
@@ -59,9 +60,15 @@ public static void firePlayerItemPickupEvent(PlayerEntity player, ItemEntity ite
         MinecraftForge.EVENT_BUS.post(new PlayerEvent.ItemPickupEvent(player, item, clone));
     }
 
+    // TODO 1.17 REMOVE
     public static void firePlayerCraftingEvent(PlayerEntity player, ItemStack crafted, IInventory craftMatrix)
```

Comment:
Same reasoning as the constructor, mark as `@Deprecated` and optionally add a small remark to ues the new method instead.

Reply:
Dne.